src/aluno/PerfilPage.tsx

```
@@ -0,0 +1,201 @@
+                const response = await UsuarioService.meuPerfil()
+
+                setUsuario(response)
+            } catch (error) {
+                console.error('Erro ao buscar dados do usuário:', error)
+            }
+        }
+
+        fetchUsuario()
+    }, [])
+
+    useEffect(() => {                                                           //Deve ser revisado após a implementação do serviço de projetos
```

ESLint

[error] Replace `····useEffect(()·=>·{·······················································` with `··useEffect(()·=>·{⏎`

(prettier/prettier)